src/Sulu/Bundle/MediaBundle/Resources/public/js/components/collections/edit/main.js

```diff
@@ -11,42 +11,27 @@ define(function() {
 
     'use strict';
 
-    var constants = {
-            mediaLanguageStorageKey: 'mediaLanguage'
-        },
-
-        namespace = 'sulu.media.collections-edit.',
-
-        /**
-         * sets the locale
-         * @event sulu.media.collections.collection-list
-         * @param {String} the new locale
-         */
-        SET_LOCALE = function() {
-            return createEventName.call(this, 'set-locale');
-        },
-
-        /**
-         * sets the locale
-         * @event sulu.media.collections.collection-list
-         * @param {Function} the callback to pass the locale to
-         */
-        GET_LOCALE = function() {
-            return createEventName.call(this, 'get-locale');
-        },
-
-        /** returns normalized event names */
-        createEventName = function(postFix) {
-            return namespace + postFix;
-        };
+    var namespace = 'sulu.media.collections.edit.',
+
+    /**
+     * emitted if the collection object has changed
+     * @event sulu.media.collections.edit.updated
+     * @param data {Object} the new collection object
+     */
+    UPDATED = function() {
+        return createEventName.call(this, 'updated');
+    },
+
+    /** returns normalized event names */
+    createEventName = function(postFix) {
```

Comment:
Yes we could (and we should). This was something i discussed with @chirimoya last year but was not implemented due to priorities